test/fstapp_SUITE.erl

```diff
@@ -0,0 +1,29 @@
+-module(fstapp_SUITE).
+
+-include_lib("common_test/include/ct.hrl").
+
+-compile(export_all).
+
+init_per_suite(Config) ->
+	Config.
+
+end_per_suite(Config) ->
+	Config.
+
+all() ->
+	[my_test_case].
+
+my_test_case(_Config) ->
+
+	application:ensure_all_started(fstapp),
```

Comment:
is a good practice starting the apps in the init_per_suite function, also we can stop it in the end_per_suite